### PR TITLE
CMQ-1487: Add V3 messaging to email templates

### PR DESCRIPTION
**Pull request by czeczotka**

### CODE REVIEW

**Comment by edjannoo** [comment]
The changes look good. We can't merge this PR until we put V3 of the spec live since once this is on master it will be rolled out to production soon afterwards. I'll hold off approving the PR until we're ready for it to be merged so that it doesn't inadvertently get merged too soon

**Review by edjannoo — APPROVED** [review]

**Review by muralishub — APPROVED** [review]

**Pull request merged**
